### PR TITLE
DASH scale requirement update

### PR DESCRIPTION
**Pull request by mgheorghe**
ENIs per 200G link reduced to 32 from 64
adjusted CPS value in HLD to match the hero test
increased flow timer from 1 sec to 5 sec ( to reduce PPS requirements for BG traffic)

### CODE REVIEW

**Review by prsunny — APPROVED** [review]
lgtm, @KrisNey-MSFT to signoff

**Comment by KrisNey-MSFT** [comment]
Will cover in DASH community call, then sign-off/merge

File: documentation/general/program-scale-testing-requirements-draft.md [hunk 1]
```diff
@@ -204,7 +204,7 @@ integrators to track and test the designs in a common manner.
         seen that meets the policy for the bi-directional flow setup.
         When using this in CPS testing, we will send a total of 6
         packets to match TCP to make things more balanced. UDP
-        bi-directional flows will be aged within 1 second after
+        bi-directional flows will be aged within 5 second after
         receiving the last packet. I would set all UDP bi-directional
```
Comment (budgrise) [inline documentation/general/program-scale-testing-requirements-draft.md:208]: Should the last sentence be removed ("0.5 to 1.0 second aging")?

File: documentation/general/program-scale-testing-requirements-draft.md [hunk 2]
```diff
@@ -228,19 +228,19 @@ integrators to track and test the designs in a common manner.
...
     -   All TCP connections should be deleted from the table after the
         test completes.
 
     -   The connection table should therefore be zero.
 
-    -   All UDP bi-directional flows need to age out before the 1 second
+    -   All UDP bi-directional flows need to age out before the 5 second
         interval to allow for new UDP bi-directional flows to be
         established. If everything works as advertised, we should never
         see the connection table go above the 8M connections. If we do
```
Comment (budgrise) [inline documentation/general/program-scale-testing-requirements-draft.md:241]: Should "8M" be replaced with a different number?

File: documentation/general/program-scale-testing-requirements-draft.md [hunk 3]
```diff
@@ -193,7 +193,7 @@ integrators to track and test the designs in a common manner.
         expect more TCP and in fact in some cases we meter UDP as a
         potential source of DoS.
 
-    -   Connection aging set to 1 sec and requires each connection or
+    -   Connection aging set to 5 sec and requires each connection or
         bi-directional flow to receive one packet every second in each
```
Comment (budgrise) [inline documentation/general/program-scale-testing-requirements-draft.md:197]: Should this say "receive one packet every 5 seconds" ?

File: documentation/general/program-scale-testing-requirements-draft.md (hunk 1 above)
Comment (mgheorghe) [reply documentation/general/program-scale-testing-requirements-draft.md:208]: corrected to 5 seconds

File: documentation/general/program-scale-testing-requirements-draft.md (hunk 2 above)
Comment (mgheorghe) [reply documentation/general/program-scale-testing-requirements-draft.md]: corrected to 32M flows or maybe it should be 15M only

File: documentation/general/program-scale-testing-requirements-draft.md (hunk 3 above)
Comment (mgheorghe) [reply documentation/general/program-scale-testing-requirements-draft.md]: corrected to 4.9 sec as long as the packet is received right before the 5s  mark  flow should remain active

File: documentation/general/dash-sonic-hld.md [hunk 4]
```diff
@@ -120,7 +120,7 @@ Following are the minimal scaling requirements
 | Active Connections/ENI        | 1M (Bidirectional TCP or UDP) |
 | Total active connections      | 32M (Bidirectional)           |
 | Metering Buckets per ENI      | 4000                          |
-| CPS                           | 1.5M                          |
+| CPS                           | 3.75M                          |
```
Comment (KrisNey-MSFT) [inline documentation/general/dash-sonic-hld.md:123]:
```suggestion
| CPS                           | 3.M                          |
```

File: documentation/general/program-scale-testing-requirements-draft.md [hunk 5]
```diff
@@ -294,19 +294,24 @@ The following scale of policies and routes are at minimum required to be
...
+| VNETs               |            | 1024        | 2048  | 4096  |  8192 |
+| ENIs                |            | 32          | 64    | 128   |  256  |
+| Outbound Routes     | 100K       | 3.2M        | 6.4M  | 12.8M | 25.6M |
+| Inbound Routes      |  10K       | 320K        | 640K  | 1.28M | 2.56M |
+| NSGs                | 5in + 5out | 320         | 640   | 1280  | 2560  |
+| ACLs prefixes       | 10x100K    | 32MM        | 64M   | 128M  | 256M  |
+| ACLs Ports          | 10x10K     | 3.2M        | 6.4M  | 12.8M | 25.6M |
+| Mappings (CA to PA) | NA[^1]     | 10M         | 20M   | 40M   | 80M   |
+| Act Con             | 1M         | 32M[^2]     | 64M   | 128M  | 256M  |
+| CPS                 |            | 3.75M       | 7.5M  | 15M   | 30M   |
```
Comment (KrisNey-MSFT) [inline documentation/general/program-scale-testing-requirements-draft.md:308]:
```suggestion
| CPS                 |            | 3M       | 6M  | 12M   | 24M   |
```

File: documentation/general/program-scale-testing-requirements-draft.md (hunk 1 above)
Comment (reshmaintel) [reply documentation/general/program-scale-testing-requirements-draft.md:208]: LGTM, thank you Mircea for changing ENI to 32 and CPS to 3M.

**Comment by KrisNey-MSFT** [comment]
Per Community Call, merging

**Pull request merged**